dangerzone/gui/main_window.py

```diff
@@ -459,38 +482,75 @@ def check_state(self) -> None:
             state = "not_installed"
 
         else:
-            # Can we run `docker image ls` without an error
+            # Can we run `docker/podman image ls` without an error
             with subprocess.Popen(
                 [container_runtime, "image", "ls"],
                 stdout=subprocess.DEVNULL,
-                stderr=subprocess.DEVNULL,
+                stderr=subprocess.PIPE,
                 startupinfo=get_subprocess_startupinfo(),
             ) as p:
-                p.communicate()
+                _, stderr = p.communicate()
                 if p.returncode != 0:
-                    log.error("Docker is not running")
+                    runtime_name = self.dangerzone.isolation_provider.get_runtime_name()
+                    log.error(f"{runtime_name} is not running")
                     state = "not_running"
+                    error = stderr
                 else:
                     # Always try installing the container
                     state = "install_container"
 
         # Update the state
-        self.state_change(state)
+        self.state_change(state, error)
 
-    def state_change(self, state: str) -> None:
+    def state_change(self, state: str, error: str) -> None:
         if state == "not_installed":
-            self.label.setText(
-                "<strong>Dangerzone Requires Docker Desktop</strong><br><br><a href='https://www.docker.com/products/docker-desktop'>Download Docker Desktop</a>, install it, and open it."
-            )
+            if platform.system() == "Linux":
+                self.label.setText(
+                    (
+                        "<strong>Dangerzone requires Podman</strong><br><br>"
+                        "Install it and retry."
+                    )
+                )
+            else:
+                self.label.setText(
+                    (
+                        "<strong>Dangerzone requires Docker Desktop</strong><br><br>"
+                        "<a href='https://www.docker.com/products/docker-desktop'>Download Docker Desktop</a>"
+                        ", install it, and open it."
+                    )
+                )
             self.buttons.show()
         elif state == "not_running":
-            self.label.setText(
-                "<strong>Dangerzone Requires Docker Desktop</strong><br><br>Docker is installed but isn't running.<br><br>Open Docker and make sure it's running in the background."
-            )
+            if platform.system() == "Linux":
+                message = (
+                    "<strong>Dangerzone requires Podman</strong><br><br>"
+                    "Podman is installed but isn't running.<br>"
+                    "It could mean that it is not configured properly."
+                )
+                if error:
+                    message += "<br><br>The following error was caught when trying to run <code>podman image list</code>"
+
+                    code_snippet = f"<pre>{error.decode()}</pre>"
```

Comment:
This may need HTML escaping.

Reply:
Thanks for the comment. The code now uses `setPlainText` instead, so no HTML is involved.